bundle/bundle.go

```diff
@@ -67,12 +67,13 @@ type LocationRef struct {
 
 // BaseImage contains fields shared across image types
 type BaseImage struct {
-	ImageType string         `json:"imageType" mapstructure:"imageType"`
-	Image     string         `json:"image" mapstructure:"image"`
-	Digest    string         `json:"digest,omitempty" mapstructure:"digest"`
-	Size      uint64         `json:"size,omitempty" mapstructure:"size"`
-	Platform  *ImagePlatform `json:"platform,omitempty" mapstructure:"platform"`
-	MediaType string         `json:"mediaType,omitempty" mapstructure:"mediaType"`
+	ImageType     string         `json:"imageType" mapstructure:"imageType"`
+	Image         string         `json:"image" mapstructure:"image"`
+	OriginalImage string         `json:"originalImage" mapstructure:"originalImage"`
```

Comment:
As far as I can remember `originalImage` is `OPTIONAL`, so I think it should be annotated with `omitempty`:
```go
OriginalImage string         `json:"originalImage,omitempty" mapstructure:"originalImage"`
```

Reply:
Correct. Fixed (with a test).